### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by F-u-n-H-o-u-s-e**
- [DEMO LINK](https://F-u-n-H-o-u-s-e.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://F-u-n-H-o-u-s-e.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,90 @@
...
+  font-size: 12px;
+}
+
+.header {
+  position: absolute;
+  left: 0;
+  top: 0;
+  margin: auto;
+  width: 1200px;
+  height: 60px;
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:25]:
```suggestion
  width: 100%;
```
do not hardcode height and width

File: src/style.css [hunk 2]
```diff
@@ -1 +1,90 @@
...
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  
+}
+
+nav {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: right;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:36]: use class selectors

File: src/style.css [hunk 3]
```diff
@@ -1 +1,90 @@
...
+
+.item {
+  padding: 22px 0 23px;
+  margin-left: 20px;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:46]: use class selectors

File: src/style.css [hunk 4]
```diff
@@ -1 +1,90 @@
...
+.navigation {
+  display: flex;
+  justify-content: right;
+}
+
+.nav__item {
+  display: block;
+  margin-left: 20px;
+  line-height: 60px;
+}*/
```
Comment (Bohdan-Maksymiuk) [inline src/style.css:90]: delete comments

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
header should be stretched in height by its content (you may use paddings for nav__items)

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Header must have 100 % width
Screenshot 2022-08-16 at 12 00 19

2.Uppercase letters for nav__list are made with styles, not hardcoded into html (you should have usual text with first uppercase letter in html)

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,49 @@
...
+      <nav class = "navigation">
+        <ul class = "table" >
+          <li class="nav__item">
+            <a class = "is-active item APPLE" href="#">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="item SAMSUNG">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="item SMARTPHONES">SMARTPHONES</a>
```
[the remark this answers is not in the document]
Comment (F-u-n-H-o-u-s-e) [reply src/index.html:25]: Made.

**Pull request closed without merging**